### PR TITLE
ERM-1257 Added tests for AgreementFilters component

### PR DESCRIPTION
**Pull request by MonireRasouli**

### CODE REVIEW

File: src/components/AgreementFilters/AgreementFilters.test.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import translationsProperties from '../../../test/helpers';
+import { activeFilters, data } from './testResources';
+import AgreementFilters from './AgreementFilters';
+
+jest.mock('@folio/stripes-erm-components', () => ({
+   ...jest.requireActual('@folio/stripes-erm-components'),
+   OrganizationSelection: () => <div>OrganizationSelection</div>,
+}));
+
+jest.mock('@folio/stripes/smart-components');
```
Comment (aditya-matukumalli) [inline src/components/AgreementFilters/AgreementFilters.test.js:16]: What does this mock do?

File: src/components/AgreementFilters/AgreementFilters.test.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        await Checkbox({ id: 'clickable-filter-agreementStatus-requested' }).click();
+        await Checkbox({ id: 'clickable-filter-agreementStatus-in-negotiation' }).click();
+        await Checkbox({ id: 'clickable-filter-agreementStatus-active' }).click();
+        await Checkbox({ id: 'clickable-filter-renewalPriority-for-review' }).click();
+        await Checkbox({ id: 'clickable-filter-renewalPriority-definitely-renew' }).click();
+        await Checkbox({ id: 'clickable-filter-renewalPriority-definitely-cancel' }).click();
+        await Checkbox({ id: 'clickable-filter-isPerpetual-yes' }).click();
+        await Checkbox({ id: 'clickable-filter-isPerpetual-no' }).click();
+        // await waitForElementToBeRemoved(() => screen.getByText('OrganizationSelection'));
+        expect(stateMock).toHaveBeenCalled();
```
Comment (aditya-matukumalli) [inline src/components/AgreementFilters/AgreementFilters.test.js:102]: The `stateMock` I believe would be called for each of the clicks above. Expecting it to be called only once doesnt guarantee if all the clicks are triggering that callback

File: src/components/AgreementFilters/AgreementFilters.test.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    test('renders the Supplementary properties Accordion', async () => {
+        await Accordion('Supplementary properties').exists();
+    });
+
+    test('renders the OrganizationSelection component', () => {
+      const { getByText } = renderComponent;
+      expect(getByText('OrganizationSelection')).toBeInTheDocument();
+    });
+
+    test('renders the checkboxes', async () => {
```
Comment (aditya-matukumalli) [inline src/components/AgreementFilters/AgreementFilters.test.js:90]: Description needs to match the action, i.e. renders vs clicking the checkboxes

**Review by aditya-matukumalli — CHANGES_REQUESTED** [review]

File: src/components/AgreementFilters/AgreementFilters.test.js (hunk 1 above)
Comment (MonireRasouli) [reply src/components/AgreementFilters/AgreementFilters.test.js:16]: It should be removed, it doesn't do anything

**Review by aditya-matukumalli — APPROVED** [review]

**Pull request merged**